Framework/Algorithms/inc/MantidAlgorithms/ReflectometryWorkflowBase2.h

```diff
@@ -39,6 +39,12 @@ class DLLExport ReflectometryWorkflowBase2
   void initMomentumTransferProperties();
   /// Initialize properties for diagnostics
   void initDebugProperties();
+  /// Initialize properties for offspec background subtraction
+  void initOffspecBgdProperties();
+  /// Initialize properties for ILL background subtraction
+  void initILLBgdProperties();
+  /// Initialize properties for SNS background subtraction
+  void initSNSBgdProperties();
```

Comment:
More generic/descriptive names here would be good based on what the method is doing rather than the instrument/facility it originated at.